images/chaos-daemon/Dockerfile

```diff
@@ -42,5 +42,7 @@ RUN case "$TARGET_PLATFORM" in \
     mv /usr/local/bin/libnsenter.so /usr/local/lib/libnsenter.so; \
     curl -L https://github.com/chaos-mesh/rs-tproxy/releases/download/v0.4.5/tproxy-$NSEXEC_ARCH-unknown-linux-gnu.tar.gz | tar xz -C /usr/local/bin
 
+RUN curl -L https://github.com/chaos-mesh/memStress/releases/download/v0.2/memStress_v0.2_linux-amd64.tar.gz | tar xz -C /usr/local/bin
```

Comment:
We should use the different download link according to different arch.

Reply:
GET. I'll add some releases for different arch.